src/services/users/index.js

```diff
@@ -123,10 +131,37 @@ const inviteUser = async (email) => {
 	return signupInfo
 }
 
+const getNewTokens = async (email, oldRefreshToken) => {
```

Comment:
its not old till a new one is generated, if you need a name for it, it can be  `currentRefreshToken` instead

Reply:
yes please